service-binding/postgresql-crunchy/src/test/java/io/quarkus/ts/sb/postgresql/OpenShiftPostgreSqlSbIT.java

```diff
@@ -0,0 +1,95 @@
+package io.quarkus.ts.sb.postgresql;
+
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.bootstrap.inject.OpenShiftClient;
+import io.quarkus.test.scenarios.OpenShiftDeploymentStrategy;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.test.utils.Command;
+
+@OpenShiftScenario(deployment = OpenShiftDeploymentStrategy.UsingOpenShiftExtensionAndDockerBuildStrategy)
+public class OpenShiftPostgreSqlSbIT {
+
+    @Inject
+    static OpenShiftClient ocClient;
+
+    @QuarkusApplication
+    static RestService app = new RestService()
+            .onPreStart(s -> createPostgresCluster());
+
+    @AfterAll
+    public static void tearDown() {
+        deleteCustomResourceDefinition("pg-cluster.yml");
+    }
+
+    /**
+     * This test verifies the application deploys successfully. If the binding fails, the application will not be
+     * available.
+     */
+    @Test
+    public void verifyPostgresServiceBoundToApplication() {
+        app.given()
+                .get("/todo/1/")
+                .then()
+                .statusCode(HttpStatus.SC_OK);
+    }
+
+    private static boolean areOperatorsInstalled() {
+        List<String> output = new ArrayList<>();
+        try {
+            new Command("oc", "get", "csv").outputToLines(output).runAndWait();
+        } catch (Exception e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+        String outputString = output.stream().collect(Collectors.joining(System.lineSeparator()));
+        return (outputString.contains("postgresoperator") && outputString.contains("service-binding-operator"));
+    }
+
+    private static void createPostgresCluster() {
+        try {
+            Thread.sleep(2000);
+        } catch (Exception e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+        Assumptions.assumeTrue(areOperatorsInstalled());
```

Comment:
@pjgg I've also added an assumption about operators being installed on the cluster so we don't have to exclude this module in Maven executions.